free-programming-books.md

```diff
@@ -1771,6 +1771,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Node.js Notes](http://books.goalkicker.com/NodeJSBook) (PDF)
 * [Node.js Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/nodejs) (PDF, Kindle) (email address *requested*, not required)
 * [The Node Beginner Book](http://nodebeginner.org)
+* [Reactive Programming with Node.js] (http://www.allitebooks.com/reactive-programming-with-node-js/)
```

Comment:
Please write the name of the author of the book after.

Remove the space between the ] and ( so it becomes a valid markdown link.